### PR TITLE
fatface.com

### PR DESCRIPTION
**Pull request by yamatt**
http://fatface.com -> https://fatface.com
http://www.fatface.com -> https://www.fatface.com

### CODE REVIEW

File: src/chrome/content/rules/Fatface.com.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<!--
+	Nonfunctional hosts in *.fatface.com:
```
Comment (J0WI) [inline src/chrome/content/rules/Fatface.com.xml]: competition.fatface.com

File: src/chrome/content/rules/Fatface.com.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+	h: http redirect
+	m: certificate mismatch
+	r: connection refused
+	s: self-signed certificate
+	t: timeout on https
+-->
+<ruleset name="Fatface.com">
+	<target host="fatface.com" />
+	<target host="www.fatface.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/Fatface.com.xml]: us.fatface.com

**Comment by jeremyn** [comment]
@yamatt Are you still interested in working on this pull request?

**Comment by yamatt** [comment]
@jeremyn what's up?

**Comment by yamatt** [comment]
Looks like the site works ok now with https-everywhere

**Comment by jeremyn** [comment]
@yamatt You're welcome to close your PR but just so you know, this site is still a valid target for HTTPS Everywhere, see https://www.eff.org/https-everywhere/faq#why-does-https-everywhere-include-rules-for-sites-like-paypal-that-already-require-https-on-all-their-pages. Generally, unless the site is HSTS preloaded ( https://hstspreload.org ) or doesn't work with HTTPS, it's valid for HTTPS Everywhere.

**Comment by yamatt** [comment]
@jeremyn I didn't know that. Is there anything extra I need to add to the PR?

It looks like https://fatface.com doesn't work anymore either.

**Comment by jeremyn** [comment]
You can try finding more subdomains using the suggestions at https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#enumerating-subdomains. @J0WI gave some examples of subdomains you missed, above.

You can also add a rewrite `rule` that rewrites `fatface.com` to `www.fatface.com`, going before the general rewrite `rule` you have. You can see an example of this [here](https://github.com/EFForg/https-everywhere/blob/cdc90c422a7a0addfefa37a744286bef7333227d/src/chrome/content/rules/16163.com.xml#L12).

**Pull request closed without merging**